types/superagent/index.d.ts

```diff
@@ -16,6 +17,8 @@ type CallbackHandler = (err: any, res: request.Response) => void;
 
 type Serializer = (obj: any) => string;
 
+type Parser = (res: request.Response, callback: (err: Error | null, body: any) => void) => void;
```

Comment:
[In browser it has other type](http://visionmedia.github.io/superagent/#parsing-response-bodies):

```typescript
type BrowserParser = (str: string) => void;
type NodeParser = (res: request.Response, callback: (err: Error | null, body: any) => void) => void;
type Parser = BrowserParser | NodeParser;
```